src/lib/public/board/data/officers.json

```diff
@@ -1,718 +1,368 @@
+      "S22": { "title": "Algo Officer", "tier": 12 },
+      "F22": { "title": "AI Team Lead", "tier": 11 }
+    }
   },
   {
     "fullName": "Johnson Tong",
-    "positions": {
-      "S21": {
-        "title": "Workshop Manager",
-        "tier": 6
-      }
-    },
-    "picture": "johnson-tong.webp"
+    "picture": "johnson-tong.webp",
+    "positions": { "S21": { "title": "Workshop Manager", "tier": 18 } }
   },
   {
     "fullName": "Alex Truong",
-    "positions": {
-      "F21": {
-        "title": "Algo Officer",
-        "tier": 15
-      }
-    },
+    "positions": { "F21": { "title": "Algo Officer", "tier": 12 } },
     "picture": "alex-truong.webp"
   },
   {
     "fullName": "Daniel Truong",
+    "picture": "daniel-truong.webp",
     "positions": {
-      "S22": {
-        "title": "Dev Project Manager",
-        "tier": 22
-      },
-      "F22": {
-        "title": "Dev Project Manager",
-        "tier": 22
-      }
-    },
-    "picture": "daniel-truong.webp"
+      "S22": { "title": "Dev Project Manager", "tier": 18 },
+      "F22": { "title": "Dev Project Manager", "tier": 18 }
+    }
   },
   {
     "fullName": "Samuel Valls",
+    "picture": "samuel-valls.webp",
     "positions": {
-      "S21": {
-        "title": "Community Manager",
-        "tier": 10
-      },
-      "F21": {
-        "title": "Create Officer",
-        "tier": 17
-      },
-      "S22": {
-        "title": "Create Operations Manager",
-        "tier": 19
-      }
-    },
-    "picture": "samuel-valls.webp"
+      "S21": { "title": "Community Manager", "tier": 22 },
+      "F21": { "title": "Create Officer", "tier": 16 },
+      "S22": { "title": "Create Operations Manager", "tier": 15 }
+    }
   },
   {
     "fullName": "Karnikaa Velumani",
+    "picture": "karnikaa-velumani.webp",
     "positions": {
-      "F21": {
-        "title": "Vice President",
-        "tier": 1
-      },
-      "S22": {
-        "title": "Vice President",
-        "tier": 1
-      },
-      "F22": {
-        "title": "President",
-        "tier": 1
-      }
-    },
-    "picture": "karnikaa-velumani.webp"
+      "F21": { "title": "Vice President", "tier": 1 },
+      "S22": { "title": "Vice President", "tier": 1 },
+      "F22": { "title": "President", "tier": 0 }
+    }
   },
   {
     "fullName": "Rina Watanabe",
-    "positions": {
-      "F21": {
-        "title": "Dev Project Manager",
-        "tier": 22
-      }
-    },
-    "picture": "rina-watanabe.webp"
+    "picture": "rina-watanabe.webp",
+    "positions": { "F21": { "title": "Dev Project Manager", "tier": 18 } }
   },
   {
     "fullName": "Jason Wong",
+    "picture": "jason-wong.webp",
     "positions": {
-      "S22": {
-        "title": "Historian",
-        "tier": 13
-      },
-      "F22": {
-        "title": "Historian",
-        "tier": 11
-      }
-    },
-    "picture": "jason-wong.webp"
+      "S22": { "title": "Historian", "tier": 9 },
+      "F22": { "title": "Historian", "tier": 9 }
```

Comment:
The updated "tier" for Historian is '10'

`"Historian": { "id": 10, "index": 600 },`